modules/serverless-functions-secrets-configmaps-manually.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-manually_{context}"]
+= Adding the configuration manually
```

Comment:
Again, more descriptive heading would be good here, and this module needs an intro sentence to explain what it does / is used for